python/dgl/graphbolt/utils/internal.py

```diff
@@ -36,9 +36,22 @@ def save_data(data, path, fmt):
 
     # Perform necessary conversion.
     if fmt == "numpy" and isinstance(element, torch.Tensor):
-        element = element.cpu().numpy()
+        element = element.cpu()
```

Comment:
what's element?

Reply:
Do we have unit test for this function?

Reply:
How should I add a new test? Do we need to create a new directory, such as `test/python/pytorch/graphbolt/utils/`?